docs/Breaking API Changes.md

```diff
@@ -87,4 +87,12 @@ All `SymbolDisplayFormat`s (predefined and user-created) now include parameter n
 
 `IncrementalGeneratorRunStep.Outputs` previously contained `IncrementalStepRunReason.Modified` as `Reason`
 when the input to the step was modified in a way that produced a new output.
-Now the reason will be reported more accurately as `IncrementalStepRunReason.New`.
\ No newline at end of file
+Now the reason will be reported more accurately as `IncrementalStepRunReason.New`.
+
+# Version 4.8.0
+
+### Changed `Assembly.Location` behavior in non-Windows
```

Comment:
Massive kudos for remembering to update the docs!

Reply:
Kudos go to @jjonescz for reminding me to do this 

https://github.com/dotnet/roslyn/pull/69406#discussion_r1286816066